File: spec/mobbnbSpec.js
'use strict';

describe("mobbnb", function(){

  var mobbnb;

  beforeEach(function(){
    mobbnb = new Mobbnb();
  });

  it("takes an email address for a user", function(){
    mobbnb.addUserEmail("mobperson@moblet.com");
    expect(mobbnb.getEmail()).toEqual("mobperson@moblet.com");
  });

  it("takes a password for a user", function(){
    mobbnb.addUserPassword("Hello1");
    // expect(mobbnb.checkPassword()).toEqual(true);
    expect(mobbnb._password).toEqual("Hello1");
  });



});

File: spec/mobbnb_spec.rb
require './lib/mobbnb'

describe Mobbnb do
  subject(:mobbnb) {Mobbnb.new}
  describe "#add_email" do
    it "allows user to add email" do
      mobbnb.add_email("mobtheworld@mob.com")
      expect(mobbnb.email).to eq("mobtheworld@mob.com")
    end
  end

  describe "#add_password" do
    it "allows user to add password" do
      mobbnb.add_password("hello1")
      expect(mobbnb.password).to eq("hello1")
    end
  end
end